showcase/src/Checkboxes/Stories.elm

```diff
@@ -30,6 +31,9 @@ update msg model =
         Checkboxes.Box3Uncheck ->
             ( { model | box3 = False }, Cmd.none )
 
+        Checkboxes.SwitchSet newValue ->
```

Comment:
If we are treating this as something different than a checkbox (UI.Switch), I would create another category in the showcase as well